content/staff-wellbeing/try-ideas-to-improve-staff-wellbeing.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Try ideas to improve staff wellbeing
+colour: blue
+---
+
+# Try ideas to improve staff wellbeing
+
+<strong class="govuk-tag">type</strong>
+
+{inset-text}
+
+## School details
+
+**School name**: King Charles I school   
+
+**Location**: Worcestershire
+
+**Phase**: Secondary
+
+**Number of pupils**: 1000
+
+**Contact details**: Email deputy headteacher Ruth Allen at <rallen@kingcharles1.worcs.sch.uk> 
+
+{/inset-text}
+
+<div class="govuk-grid-row dfe-width-container">
+  <div class="govuk-grid-column-full">
+    <div class="info-box">
+      <div class="info-box__corner">
+        <img src="/assets/images/bullseye.svg" alt="Bullseye icon">
+      </div>
+      <h2 class="govuk-heading-m">
+        Impact and outcomes
+      </h2>
+      <p>
+        We made these changes to enable every teacher to become highly effective by:
+        <ul>
+          <li>
+            ensuring every teacher has time to focus on what is important - planning, teaching and feedback
+          </li>
+          <li>
+            believing in simplicity, always taking the shortest route and aiming for maximum impact on student learning with minimal workload for staff
+          </li>
+          <li>
+            continuously reviewing and evaluating our systems in order to support all staff to achieve a healthy work life balance
+          </li>
+        </ul>
+      </p>
+    </div>
+  </div>
+</div>
+
+## Background from Ruth Allen, Deputy Headteacher
+
+{inset-text}
+
+At King Charles I school we know that teaching is exhilarating and rewarding. But we also recognise that it can be exhausting.  We understand that time is precious and that tired teachers do not make effective teachers. Our aim is for staff to not have to take work home. 
+
+{/inset-text}
+
+## 21 things we did to improve staff wellbeing 
+
+1. Removal of ‘potential cover’ from timetables except for staff who are under their allocated number of lessons. 
+
+2. Removal of pointless, time wasting displays, we are currently in the process of removing all display boards from classrooms and corridors. 
+
+3. No time wasted on routine admin tasks, departments such as reprographics, finance and data are all on hand to support teachers.  
+
+4. Weekly admin support for heads of department.  
+
+5. We use ParentMail and ParentPay to ensure that tutors are not required to collect money or reply slips. 
+
+6. Automated rewards and simple one click recording of achievement points.   
+
+7. Member of SLT responsible for staff workload and wellbeing. 
+
+8. Half termly staff forum so that staff can highlight issues and concerns and we can move the school forward together. 
```

Comment:
```suggestion
8. A half-termly staff forum takes place so that staff can highlight issues and concerns, and we can move the school forward together. 
```